modules/sql/src/main/resources/db/migration/V059__Niri_KeywordNames.sql

```diff
@@ -0,0 +1,47 @@
+---------------------------
+-- NIRI FITS Keyword Names
+---------------------------
+
+INSERT INTO e_fits_keyword_names(id, name)
```

Comment:
Are keywords for all the instruments going to be put into the same table?

Reply:
Yes, but my plan is to have the definitions in separate files, one per instrument.